crates/common_enums/src/enums.rs

```diff
@@ -41,6 +41,11 @@ pub enum CountryCode {
     US
 }
 
+#[derive(Clone, Copy, Debug, Serialize, Deserialize)]
+pub enum AbsentImpl {
+    ManualCapture,
+}
```

Comment:
can we have it in snake case? it stands out quite a bit in the configuration file